src/run.ts

```diff
@@ -211,8 +207,8 @@ export async function run() {
             throw Error("Unknown render engine");
     }
 
-    var isSilent = true;
-    var silentInput = core.getInput('silent', { required: false });
+    let isSilent = true;
```

Comment:
Just do:

const isSilent = core.getInput('silent') === 'false';

The three = checks for type equality which allows for it to be condensed this way.

Core.getInput defaults to not required so this just cuts down some unnecessary things.